force_wfmanager/central_pane/plot.py

```diff
@@ -75,6 +75,13 @@ def __data_arrays_default(self):
     def _get__value_names(self):
         return self.analysis_model.value_names
 
+    @on_trait_change('_value_names')
```

Comment:
One possible solution is that you invoke_later the change... but maybe there's a more elegant one. I'll think about it.